README.md

```diff
@@ -28,19 +28,19 @@ A command-line benchmarking tool.
 
 To run a benchmark, you can simply call `hyperfine <command>...`. The argument(s) can be any
 shell command. For example:
-``` bash
+```sh
```

Comment:
what is the advantage?

Reply:
No bash extensions used – indicate that clearly and in contrast to the parts where they are.